underlay/src/main/java/bio/terra/tanagra/underlay/IndexSchema.java

```diff
@@ -206,6 +225,15 @@ private static void fromConfigEntity(
     entityLevelDisplayHintTables.add(
         new ITEntityLevelDisplayHints(nameHelper, szBigQueryIndexData, szEntity.name));
 
+    // EntitySearchByAttribute tables.
+    if (szEntity.optimizeSearchByAttributes != null) {
+      szEntity.optimizeSearchByAttributes.forEach(
+          attribute ->
```

Comment:
This isn't updated to match the config change, is it? Shouldn't this be a list now?

Reply:
correct. some of the changes are not pushed yet